src/window.js

```diff
@@ -136,7 +139,6 @@ class Window extends Adw.ApplicationWindow {
   #updateWebView = () => {
     this._webview = this._tab_view.selected_page.child;
     this.#updateButtons();
-    this.title = this._webview.title;
```

Comment:
The window's title was being updated here before, but it's probably better to do it with ``binding_property_full`` like I mentioned above to handle null values ig

Reply:
I think it was clean enough as it was, but adding the sanity check and setting the navigation page title too. Done that.